models/quality_measures/intermediate/nqf2372_breast_cancer_screening/quality_measures__int_nqf2372_exclude_advanced_illness.sql

```diff
@@ -154,19 +130,6 @@ with denominator as (
 
 )
 
-, observation_exclusions as (
-
-    select
-          observations.patient_id
-        , observations.observation_date
-        , exclusion_codes.concept_name
-    from observations
-         inner join exclusion_codes
-             on observations.code = exclusion_codes.code
-             and observations.code_type = exclusion_codes.code_system
-
-)
-
```

Comment:
Removed observation logic. This doesn't work correctly when we need to look for both Acute Inpatient and Advanced Illness. We can revisit at another time.